jdbc-v2/src/main/java/com/clickhouse/jdbc/ConnectionImpl.java

```diff
@@ -484,7 +484,7 @@ public Properties getClientInfo() throws SQLException {
     public Array createArrayOf(String typeName, Object[] elements) throws SQLException {
         try {
             // TODO: pass type name
-            return new com.clickhouse.jdbc.types.Array(List.of(elements), Types.OTHER);
+            return new com.clickhouse.jdbc.types.Array(List.of(elements), "", Types.OTHER);
```

Comment:
I think we can determine type of the elements here and map it to certain types. 
JdbcUtils has such mapping alread.